onnxruntime_extensions/util.py

```diff
@@ -112,11 +112,13 @@ def remove_unused_initializers(subgraph, top_level_initializers=None):
     all_initializers = initializers + top_level_initializers
 
     # Filter the initializers by checking if their names are in the list of used input tensors
-    used_initializers = [init for init in all_initializers if init.name in input_tensors]
+    used_initializers = [
+        init for init in all_initializers if init.name in input_tensors]
 
     # Update the subgraph's initializers
     del subgraph.initializer[:]
-    subgraph.initializer.extend([init for init in used_initializers if init in initializers])
+    subgraph.initializer.extend(
+        [init for init in used_initializers if init in initializers])
```

Comment:
for all the indentation/line changes, are you using a specific standard (ex: 80 chars max per line)? i can also make sure to do this in the future so we can minimize such cleanup in the future.

Reply:
it's not about the maximum line length, I think some AI tools are trying to beautify the code and make it be more readable.

Reply:
sounds good!